bitnami/keycloak/25/debian-12/rootfs/opt/bitnami/scripts/libkeycloak.sh

```diff
@@ -198,7 +198,7 @@ keycloak_configure_health_endpoints() {
 keycloak_configure_hostname() {
     info "Configuring hostname settings"
     ! is_empty_value "$KEYCLOAK_HOSTNAME" && keycloak_conf_set "hostname" "${KEYCLOAK_HOSTNAME}"
-    keycloak_conf_set "hostname-strict" "false"
```

Comment:
Per [docs](https://www.keycloak.org/server/hostname):
>Should always be set to true in production, unless your reverse proxy overwrites the Host header. If enabled, the hostname option needs to be specified.

This option should be `true` by default. The user can override this using `KC_HOSTNAME_STRICT` if necessary.

Reply:
I realized that chart supports empty hostname; I added another variable to define this value via chart.